docs/ml-features.md

```diff
@@ -1093,13 +1111,10 @@ for more details on the API.
 
 `QuantileDiscretizer` takes a column with continuous features and outputs a column with binned
 categorical features.
-The bin ranges are chosen by taking a sample of the data and dividing it into roughly equal parts.
-The lower and upper bin bounds will be `-Infinity` and `+Infinity`, covering all real values.
-This attempts to find `numBuckets` partitions based on a sample of the given input data, but it may
-find fewer depending on the data sample values.
+The bin ranges are chosen using the `approxQuantile` method based on the Greenwald-Khanna algorithm.
```

Comment:
See []() - I think we can say something like

```
The bin ranges are chosen using an approximate algorithm (see the documentation for approxQuantile for a detailed description).
```

We could link the the `approxQuantile` to the relevant API doc link.